packages/rancher-logging/templates/crd-template/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-version: 3.14.0
```

Comment:
I still see this version in the assets and charts folders. Assuming its still in the index as well.

Reply:
It also still in the 'new charts' issue